docker/Dockerfile.collector

```diff
@@ -4,16 +4,17 @@
 
 
 ########
-FROM golang:1.20-bullseye as backendbuild
+FROM golang:1.20-bookworm as backendbuild
 
 WORKDIR /go/src/github.com/analogj/scrutiny
 
 COPY . /go/src/github.com/analogj/scrutiny
 
+RUN apt-get update && apt-get install -y file
```

Comment:
what is `file` used for?

Reply:
we should cleanup the `apt/lists` after this package is installed:

https://docs.docker.com/develop/develop-images/instructions/#apt-get

Reply:
During make, file seems to be used to identify the binary. For what purpose, I don't know. To get rid of the error though I install file. It can probably be removed afterwards.